rune/Makefile

```diff
@@ -13,6 +13,8 @@ BUILDTAGS ?= seccomp selinux apparmor
 COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
 COMMIT ?= $(if $(shell git status --porcelain --untracked-files=no),"$(COMMIT_NO)-dirty","$(COMMIT_NO)")
 VERSION := $(shell cat ./VERSION)
+INCLAVARE_CONTAINERS_VERSION := $(shell cat ../VERSION)
```

Comment:
So if rune is available on a separate repo, how to acquire INCLAVARE_CONTAINERS_VERSION? You know ../VERSION will be not available then.